### PR TITLE
Meta key support

### PR DESCRIPTION
**Pull request by nkt**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (AsaAyers) [inline .editorconfig]: What is this? It probably doesn't need to be committed.

File: lib/code-links.js [hunk 2]
```diff
@@ -31,8 +31,8 @@ module.exports = {
         modifierKey: {
             description: 'Hold this key and click a link to open it',
             type: 'string',
-            default: (process.platform === 'darwin') ? 'alt' : 'ctrl',
-            enum: [ 'ctrl', 'alt' ]
+            default: (process.platform === 'darwin') ? 'meta' : 'ctrl',
```
Comment (AsaAyers) [inline lib/code-links.js]: There is already a default established. Please don't just go changing it.

File: lib/code-links.js [hunk 3]
```diff
@@ -31,8 +31,8 @@ module.exports = {
...
+            enum: [ 'ctrl', 'alt', 'meta' ]
         },
         displayMode: {
             type: 'string',
```
Comment (AsaAyers) [inline lib/code-links.js:38]: Lines 51/52 also need to account for the new `code-links-meta` class.

File: keymaps/code-links.cson [hunk 4]
```diff
@@ -2,7 +2,14 @@
...
\ No newline at end of file
+'body.code-links-ctrl atom-text-editor': {
+  'ctrl': 'code-links:tmp-show-links'
+}
+
+'body.code-links-alt atom-text-editor': {
+  'alt': 'code-links:tmp-show-links'
+}
+
+'body.code-links-alt atom-text-editor': {
```
Comment (AsaAyers) [inline keymaps/code-links.cson]: `body.code-links-meta`

File: .editorconfig (hunk 1 above)
Comment (nkt) [inline .editorconfig]: Actually it's needed, cause you are using windows `end_of_line`, I'm unix.
This file specify this. This file also helps other people that would to contribute.
http://editorconfig.org/

Comment (AsaAyers) [inline .editorconfig]: and editorconfig is picked up by Atom automatically? Also, windows line endings are stupid. I'll convert the whole repo now.

Comment (nkt) [inline .editorconfig]: Not automaticly. But there are plugins for every editor/IDE, and this plugins are popular.
https://atom.io/packages/editorconfig

Comment (nkt) [inline .editorconfig]: This file specify very simple, but really needed rules, like:
- indent using tabs or spaces
- indent size
- charset
- end of line style
- should editor insert final newline
- should editor removes whitespaces

Comment (AsaAyers) [inline .editorconfig]: I'd still rather not clutter the project with 3rd party files unrelated to the project. I just went through normalizing the files and it seems like `keymaps/code-links.cson` is the only thing that had CRLF line endings.

I use https://atom.io/packages/project-manager and it does this and more.

**Comment by nkt** [comment]
Great. I've broken patch.

**Comment by AsaAyers** [comment]
Sorry, that's my fault. I just removed the CRLFs in #15

**Comment by AsaAyers** [comment]
Something has gone wrong here, github now says it's a patch from `unknown repository`. You can add `.diff` to the end of PRs (https://patch-diff.githubusercontent.com/raw/AsaAyers/code-links/pull/14.diff). I'm just going to apply the diff and call it done. No need to figure out why the PR isn't cooperating.

**Comment by nkt** [comment]
Thanks!

**Comment by AsaAyers** [comment]
I just pushed this and the underline color as 0.3.10

**Pull request closed without merging**